src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3203,7 +3203,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       AssignTargetPaths;
       SplitResourcesByCulture;
       CreateManifestResourceNames;
-      CreateCustomManifestResourceNames
+      CreateCustomManifestResourceNames;
+      AssignEmbeddedResourceOutputPaths;
```

Comment:
Changing dependencies is scary but since this is only adding a new target I'm not too concerned.